app/cli/cmd/attestation_add.go

```diff
@@ -47,6 +47,16 @@ func newAttestationAddCmd() *cobra.Command {
 		Annotations: map[string]string{
 			useWorkflowRobotAccount: "true",
 		},
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			switch {
+			case name == "" && kind == "":
+				return fmt.Errorf("both --name and --kind cannot be empty")
+			case name != "" && kind != "":
+				logger.Warn().Msg("both --name and --kind are provided, --kind will be ignored")
```

Comment:
I personally lean towards rapid and clear user feedback. In this case I think it would be more useful to fail.